corehq/apps/app_manager/xform.py

```diff
@@ -543,7 +545,11 @@ def validate_xform(source):
         source = source.encode("utf-8")
     # normalize and strip comments
     source = ET.tostring(parse_xml(source))
-    validation_results = formtranslate.api.validate(source)
+    try:
+        validation_results = nimbus_api.validate_form(source)
+    except NimbusAPIException:
+        return
```

Comment:
Is it possible to give a strictly informational warning message (that does not prevent those other actions)? It seems bad to have "could not validate" look the same as "your form is valid."